.eslintrc.js

```diff
@@ -103,6 +103,7 @@ module.exports = {
 
     // ERRORS OF plugin:react/recommended
     'react/no-unescaped-entities': 'off',
+    'react/display-name': 'off',
```

Comment:
@GiladShoham please review this change.
We added this rule in a few files, so better to add this globally.